tests/integration/test_transfer.py

```diff
@@ -0,0 +1,8 @@
+from ape import accounts, networks
+
+
+def test_transfer():
```

Comment:
Originally had put this in a script and put it in the projects, but that seemed weird... Because, here, I just want to test that `account.transfer()` works (in an integration-test sort of way), not that `ape run` works.

Reply:
Agree